### DISCUSSION TITLE
[Crash] "The software closed because an error occurred" shortly after attempting to launch game with arcropilis

### DISCUSSION THREAD

**Discussion by RyufromStreets** [Raytwo/ARCropolis#319] [Issues] [answered]
Posted this where it didn't belong previously, I'll just copy and paste most of what I had there
-----
**System Firmware Version**

14.1.0
**Atmosphere Version**

1.3.1
**Smash Version**

13.0.1
**ARCropolis Version**

3.3.0
**Version channel**

Stable
**SD card formatting**

FAT32
**Can the crash be consistently reproduced?**

Yes
**Configuration**

 Irrelevant/Not provided

**At what moment does the crash happen?**

After the Switch boot animation

 **Details**

I'm trying to update to Acropolis 3.3.0 (Arcropolis 3.0.0 beta 3.1 was the version I was running previously, I didn't run into many issues with it).
If I do a completely clean installation of acropolis I can successfully get past the configuration menu, but any attempt to relaunch smash ends up in failure whenever I try to boot with arcropolis, leaving no log report. Holding L to stop arcropolis from working allows for the game to boot up normally.
I've attempted

-     Doing a clean installation multiple times
-     Deleting all mods from my sd card
-     Fix Archive Bit didn't show any errors
-     Ran a check disk from windows and that didn't find any errors either

Honestly I'm not sure what else I could do to resolve this on my own, the other places I've asked haven't been able to provide an answer. Is there some other folder for arcropolis that I'm supposed to be deleting for a fresh install? I've deleted the "01006A800016E000" and the "ultimate" entirely before installing. Not sure what else I could do.

**Crash report**

Atmosphère Crash Report (v1.7):
Result: 0x4A8 (2168-0002)

Process Info:
Process Name: Application
Program ID: 01006a800016e000
Process ID: 000000000000008e
Process Flags: 00000077
User Exception Address: 0000000bf614b000
Exception Info:
Type: Data Abort
Address: 0000000000000000
Crashed Thread Info:
Thread ID: 00000000000002c1
Thread Name: Thread_0x0000005BF5C9E670
Registers:
X[00]: 0000000000000000
X[01]: 0000000000000000
X[02]: 0000005bf5783c98
X[03]: 00000038c02f0050
X[04]: 0000000000000078
X[05]: 0000000000000000
X[06]: 0000000000000001
X[07]: 0000000000000000
X[08]: 0000000000000000
X[09]: 0000000000000000
X[10]: 000000000000000f
X[11]: 0000005bf57839d6
X[12]: 0000000000003432
X[13]: 0000000000000000
X[14]: 0000000000000000
X[15]: 0000005bf5783b88
X[16]: 0000005bf63aedc8
X[17]: 0000005bf5c63070 (nnSdk + 0x4de070) (__memcpy_device + 0x0)
X[18]: 0000005bf6225c38
X[19]: 00000038c2d67880
X[20]: 0000000bf614b148
X[21]: 0000005bf5784000
X[22]: 0000005bf5784000
X[23]: 00000038c1fc0000
X[24]: 0000000000000000
X[25]: 0000000000000001
X[26]: 000000245c97edd1
X[27]: 0000000000000000
X[28]: 0000000000000000
FP: 000000245c97ecc0
LR: 0000005bf5764188 (Skyline + 0x5188)
SP: 0000005bf5783be0
PC: 0000005bf5764188 (Skyline + 0x5188)
Stack Trace:
ReturnAddress[00]: 0000005bf597aafc (nnSdk + 0x1f5afc) (_ZN2nn2fs9WriteFileENS0_10FileHandleElPKvmRKNS0_11WriteOptionE + 0xdc)
ReturnAddress[01]: 000000519a22e0f8 (arcropolis + 0x1690f8)
ReturnAddress[02]: 0000005bf5ab94b8 (nnSdk + 0x3344b8)
ReturnAddress[03]: 0000005bf5abd984 (nnSdk + 0x338984)
ReturnAddress[04]: 0000000000000000
TLS Address: 0000001af0790400
TLS Dump: 00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f
001af0790400 06 00 00 00 08 00 00 00 00 00 00 00 00 00 00 00
001af0790410 02 00 00 00 8a 00 00 00 00 00 00 00 00 00 00 00
001af0790420 53 46 43 4f 00 00 00 00 00 00 00 00 a1 6d 71 f3
001af0790430 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790440 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790450 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790460 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790470 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790480 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af0790490 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904a0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904b0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904c0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904d0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904e0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
001af07904f0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
Module Info:
Number of Modules: 7
Module 00:
Address: 0000005bedbdc000-0000005bedc00000
Name: [00000000]
Module Id: 0000000000000000000000000000000000000000000000000000000000000000
Module 01:
Address: 0000005bedc00000-0000005bedc02000
Name: nnrtld
Module Id: 285183CAC95179371CC22CEC3723F882E1589659000000000000000000000000
Module 02:
Address: 0000005bedc04000-0000005bf15cb000
Name: cross2_Release.nss
Module Id: B9B166DF1DB90BAFFBD8027EB0DF14D6949D6A11000000000000000000000000
Module 03:
Address: 0000005bf504a000-0000005bf53b7000
Name: multimedia
Module Id: 60C6EDBBFCEBD4176CC301D5D9CB5A6F094F9360000000000000000000000000
Module 04:
Address: 0000005bf575f000-0000005bf5778000
Name: Skyline
Module Id: B44280982D6BBFA90259E041B53C1889D3F10328000000000000000000000000
Module 05:
Address: 0000005bf5785000-0000005bf5cc6000
Name: nnSdk
Module Id: 5FD4F531C1195E9E5510C9E73F22CD39612839F8000000000000000000000000
Module 06:
[01650486583_01006a800016e000.log]
(https://github.com/Raytwo/ARCropolis/files/8525738/01650486583_01006a800016e000.log

**Comment by RyufromStreets** [comment]
Update: I wasn't able to find out what was going wrong, but I was able to resolve the issue via just straight up uninstalling and re-downloading smash (maybe there was a corrupt game or update file, I have no way of knowing). Adding my mods back all at once caused the game to crash on the nintendo switch boot screen again, I had to add back all of my mods in chunks but now thats over with and I haven't experienced any crashes after launching the game. So it seems that's the end of it.

**Comment by RyufromStreets** [comment]
Update: The crash has begun rearing its ugly head again so it seems like reinstalling smash is not a permanent solution. So I'm kinda back a square one, although I did manage to get some log files from when the crashes began to happen.
This one was from the third crash i experienced going into a training mode match. I did not have log to file enabled for the previous two.
[2022-04-22_05-24-41.log](https://github.com/Raytwo/ARCropolis/files/8537508/2022-04-22_05-24-41.log)

This one i got after doing a fresh install of arcropolis (I left my mods on the SD card this time, all the mods that are creating conflicts would have normally been disabled)

[2022-04-22_05-45-53.log](https://github.com/Raytwo/ARCropolis/files/8537528/2022-04-22_05-45-53.log)

I'm going to try deleting smash again and enable logging to file from the start and see where that gets me

**Comment by Raytwo** [comment]
The crash report you posted in your opening message seems to indicate that something couldn’t be properly written.

Try running the archive bit fix with something like Hekate maybe?

**Reply by RyufromStreets** [reply]
Fix archive bit turn up with "Total archive bits fixed: 0 unset, 0 set!"
which I'm guessing means it found nothing wrong, it was the same result the last time i tried it.

I did another surface test and there are no errors there as well
![read test](https://user-images.githubusercontent.com/104103776/164785504-078799ae-6625-4962-b173-3a0724a78d1c.PNG)

I just don't get it, everything turns up fine on a fresh acropolis/skyline install. All the non conflicting mod files loaded up normally as far as I could tell, but every reboot ends in failure barring the first time I reinstalled smash where things seemed fine for a few hours.

**Comment by RyufromStreets** [comment, marked as answer]
Last update most likey: I decided to return to Arcro 3.0.0 beta 3.1
I can't figure out what's wrong and even if I did I'd have no way of knowing how to fix it. I'll stick to using this version of arcropolis
